ironfish/src/rpc/routes/wallet/exportAccount.ts

```diff
@@ -5,11 +5,12 @@ import * as yup from 'yup'
 import { ApiNamespace, router } from '../router'
 import { getAccount } from './utils'
 
```

Comment:
I know you didn't introduce this change, but why is `account?` possibly undefined? Shouldn't we always requested account always have a value? I suppose this might be to handle export of default acccount, but it seems like that might be buried in the RPC somewhere in `getAccount`. Might be worth it to make it more explicit.

Reply:
Yeah, if no `account` is defined, the default is account is exported, exactly.